apps/worker/src/app/workflow/workflow.module.ts

```diff
@@ -110,14 +110,6 @@ const SERVICES: Provider[] = [
     provide: MetricQueueService,
     useClass: MetricQueueService,
   },
-  {
-    provide: QueueService,
-    useClass: WorkflowQueueService,
-  },
-  {
-    provide: TriggerQueueService,
-    useClass: TriggerProcessorQueueService,
-  },
```

Comment:
I think this didn't make sense. Might be very wrong.
CC: @davidsoderberg ?

Reply:
@p-fernandez these are injected to the `BULLMQ_LIST` in line 120... also please check all the places where `QueueService` and `TriggerQueueService` were used before and change there accordingly... from what I remember is that `AddJob` use-case is using `private queueService: QueueService` in the constructor because the `WorkflowQueueService` is not the part of the `application-generic` package